src/Workspaces/Core/Portable/Shared/Collections/IntervalTree`1.cs

```diff
@@ -17,6 +17,7 @@ internal partial class IntervalTree<T> : IEnumerable<T>
         public static readonly IntervalTree<T> Empty = new IntervalTree<T>();
 
         protected Node root;
+        private readonly int count;
```

Comment:
This is for when you're in the debugger and just trying to do any sort of EE is super unstable.  Sometimes a field is nice and reliable.  I will comment this though.